TWLight/resources/models.py

```diff
@@ -439,6 +439,14 @@ def clean(self):
             raise ValidationError(
                 "Partners with automatically sent messages require user instructions to be entered"
             )
+        if (
+            self.authorization_method == self.PROXY
+            or self.authorization_method == self.BUNDLE
+        ):
```

Comment:
One additional thought I had, which wasn't in the original Phabricator ticket (sorry!), is that we should also check in this if statement that the resource doesn't have streams. If there are streams the streams have `target_url` instead of the resource. This isn't obvious and is a bad design decision from us, but it is the way it is :)

This would just mean adding a `and not self.specific_stream` here. I think it would also be helpful to add another test for this case.